pkg/server/node.go

```diff
@@ -114,6 +115,20 @@ This metric is thus not an indicator of KV health.`,
 		Measurement: "Disk stalls detected",
 		Unit:        metric.Unit_COUNT,
 	}
+
+	metaInternalBatchRPCMethodCount = metric.Metadata{
+		Name:        "internal.batch.rpc.%s.recv",
+		Help:        "Number of %s requests processed",
+		Measurement: "RPCs",
+		Unit:        metric.Unit_COUNT,
+	}
+
+	metaInternalBatchRPCCount = metric.Metadata{
+		Name:        "internal.rpc.batches",
+		Help:        "Number of batches processed",
+		Measurement: "Batches",
+		Unit:        metric.Unit_COUNT,
```

Comment:
What's the motivation for the `internal.` prefix? Seems like just using a `rpc.` prefix would suffice? In that case we should put the method counts under a separate group though, something like `rpc.method.%s.recv` or `rpc.request.%.recv`.

Aside from the bikeshedding, LGTM -- thanks for adding these!

Reply:
I originally had `rpc.` but added `internal.` thinking that adding the service would be more clear. Now that you mention it, it seems redundant. Thanks for the feedback!